Improve descriptions tag (#2772)
near/docs: docs/tools/sdk.md

id: sdk
title: NEAR SDK
hide_table_of_contents: false

> description: "Learn NEAR SDK for building smart contracts in Rust and JavaScript with examples and references."

Reviewer: ```suggestion
description: "Choose an SDK to start building contracts."
```
